contrib/checkbox-ce-oem/checkbox-provider-ce-oem/bin/gst_transform_resize.py

```diff
@@ -0,0 +1,253 @@
+        "-ep",
+        "--encoder_plugin",
+        required=True,
+        type=str,
+        help="Encoder plugin be used in gstreamer pipeline e.g. v4l2h264enc",
+    )
+
+    parser.add_argument(
+        "-wf",
+        "--width_from",
+        type=str,
+        required=True,
+        help="Value of width of the origianl resolution",
+    )
+
+    parser.add_argument(
+        "-hf",
+        "--height_from",
+        type=str,
+        required=True,
+        help="Value of height of the origianl resolution",
+    )
+
+    parser.add_argument(
+        "-wt",
+        "--width_to",
+        type=str,
+        required=True,
+        help="Value of width of the target resolution",
+    )
+
+    parser.add_argument(
+        "-ht",
+        "--height_to",
+        type=str,
+        required=True,
+        help="Value of height of the target resolution",
+    )
+
+    parser.add_argument(
+        "-f",
+        "--framerate",
+        type=str,
+        default="",
+        help="Value of framerate. e.g. 60, 30",
+    )
+
+    args = parser.parse_args()
+    return args
+
+
+def project_factory(args: argparse.Namespace) -> Any:
+    """
+    Factory function to create a project instance based on the platform
+    specified in the argparse arguments.
+    Args:
+        args (argparse.Namespace): A parsed argument object that contains the
+            project parameters.
+    Returns:
+        Any: An instance of the project class (e.g., `GenioProject`) created
+            with the specified parameters.
+    Raises:
+        SystemExit: If the platform is not recognized or supported.
+    """
+    if "genio" in args.platform:
+        return GenioProject(
+            platform=args.platform,
+            codec=args.encoder_plugin,
+            width_from=args.width_from,
+            height_from=args.height_from,
+            width_to=args.width_to,
+            height_to=args.height_to,
+            framerate=args.framerate,
+        )
+    else:
+        raise SystemExit(
+            "Error: Cannot get the implementation for '{}'".format(
+                args.platform
+            )
+        )
+
+
+class GenioProject(PipelineInterface):
+    """
+    Genio project manages platforms and codecs, and handles
+    building.
+    Spec: https://download.mediatek.com/aiot/download/release-note/v24.0/v24.0_IoT_Yocto_Feature_Table_v1.0.pdf     # noqa: E501
+    """
+
+    def __init__(
+        self,
+        platform: str,
+        codec: str,
+        width_from: int,
+        height_from: int,
+        width_to: int,
+        height_to: int,
+        framerate: int,
+    ):
+        self._platform = platform
+        self._codec = codec
+        self._width_from = width_from
+        self._height_from = height_from
+        self._width_to = width_to
+        self._height_to = height_to
+        self._framerate = framerate
+        self._codec_parser_map = {
+            GStreamerEncodePlugins.V4L2H264ENC.value: "h264parse"
```

Comment:
Is `V4L2H265ENC` no longer used in this test case? I remember seeing it before

Reply:
Yes, I only care `V4L2H264ENC` in this scenario